pkg/apis/serving/v1/revision_helpers.go

```diff
@@ -60,14 +61,14 @@ const (
 	// RoutingStatePending is a state after a revision is created, but before
 	// its routing state has been determined. It is treated like active for the purposes
 	// of revision garbage collection.
-	RoutingStatePending RoutingState = "pending"
+	RoutingStatePending RoutingState = "Pending"
```

Comment:
I wanted to ask what the typical practice is with annotations and capitalization in k8s?